e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -344,6 +348,12 @@ describe("scenarios > setup", () => {
       cy.findByLabelText("Hallo, John. Leuk je te ontmoeten!").should(
         "be.visible",
       );
+
+      if (IS_ENTERPRISE) {
+        cy.button("I'll activate later").click();
```

Comment:
We did not localize `I'll activate later` in French.

Reply:
Whoops, sorry! I have no idea what language this is hahaha

Reply:
Fixed in https://github.com/metabase/metabase/pull/63577/commits/4fdba8468a9e5efdc1bb0094cbcc8a011da7b90e, I think it might have already been localized but my local uberjar isn't up to date 🙏🏻